README.md

```diff
@@ -16,19 +16,49 @@
   <br><br>
   <a href="https://github.com/parallaxsecond/parsec/actions?query=workflow%3A%22Continuous+Integration%22"><img src="https://github.com/parallaxsecond/parsec/workflows/Continuous%20Integration/badge.svg" alt="CI tests"/></a>
   <a href="https://travis-ci.com/parallaxsecond/parsec"><img src="https://travis-ci.com/parallaxsecond/parsec.svg?branch=master" alt="Travis CI tests"/></a>
   <a href="https://opensource.org/licenses/Apache-2.0"><img src="https://img.shields.io/badge/License-Apache%202.0-blue.svg" alt="License"/></a>
 </p>
 
-**Parsec** is the **P**latform **A**bst**R**action for **SEC**urity, a new open-source initiative
-to provide a common API to secure services in a platform-agnostic way.
+# Welcome To PARSEC
 
-Read the Parsec documentation [online](https://parallaxsecond.github.io/parsec-book/)!
+**PARSEC** is the **P**latform **A**bst**R**action for **SEC**urity, a new open-source initiative
+to provide a common API to hardware security and cryptographic services in a platform-agnostic way. This abstraction layer keeps workloads decoupled from physical platform details, enabling cloud-native delivery flows within the data center and at the edge.
 
-## Disclaimer
+<p align="center">
+  <img src="./doc/images/parsec/ParsecConceptDiagram.png" alt="Parsec Concept Diagram"/>
+</p>
+
+Read the Parsec documentation [online](https://parallaxsecond.github.io/parsec-book/).
+
+Read the whitepaper [Security for the Infrastructure Edge](https://www.arm.com/-/media/global/people/Security-For-The-Infrastructure-Edge-White-paper-NDA).
+
+**Note**: Arm is contributing to Parsec as part of [Project Cassini](https://www.arm.com/-/media/global/solutions/artificial-intelligence/Project_Cassini.pdf).
+
+# Why PARSEC?
+
+Use Parsec when you need:
+
+- A **portable interface to your platform's Root of Trust** in order to manage keys and perform cryptographic operations without knowledge of the hardware.
+- A simple and portable way to access the **best available security** of your platform in your **preferred programming language**.
+
+# What PARSEC Provides
+
+The value proposition of Parsec is that it provides the following:
+
+- **Abstraction** – a common API that is truly agnostic and based on modern cryptographic principles, grounded in the [Platform Security Architecture](https://www.arm.com/why-arm/architecture/platform-security-architecture)
+- **Mediation** – security as a microservice, brokering access to the hardware and providing isolated key stores in a multi-tenant environment
+- **Ergonomics** – a client library ecosystem that brings the API to the fingertips of developers in any programming language: “easy to consume, hard to get wrong”
+- **Openness** – an open-source project inviting contributions to enhance the ecosystem both within the service and among its client libraries
+
+# Partner Organizations
+
+See who is [using and contributing to PARSEC](./PARTNERS.md).
```

Comment:
This does indeed sound like a replacement for [`CONTRIBUTORS.md`](https://github.com/parallaxsecond/parsec/blob/master/CONTRIBUTORS.md)

Reply:
The PARTNERS file is different because it's intended to be a place where we put logos for organisations who are either contributing to or adopting/integrating the PARSEC technology in their deployments. This is non-obvious at the moment, because the file has no content.